src/neo/ProtocolSettings.cs

```diff
@@ -33,10 +34,19 @@ public static ProtocolSettings Default
             {
                 if (_default == null)
                 {
-                    var configuration = Utility.LoadConfig("protocol");
+                    IConfigurationRoot configuration;
+                    try
+                    {
+                        configuration = Utility.LoadConfig("protocol");
+                    }
+                    catch (System.Exception e)
+                    {
+                        Console.WriteLine("Failed parsing protocol.json. Using default settings");
+                        Console.WriteLine(e.ToString());
```

Comment:
Don't use `Console.WriteLine()` in a non-console project.

Reply:
@KickSeason  plz use `Utility.Log`